week-3/D-array-filter/exercise.js

```diff
@@ -7,8 +7,12 @@
 */
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
-
-var pairsByIndex; // Complete this statement
+function checkingValue(value) {
+  if (value instanceof Array && value.length >= 2) {
```

Comment:
More than 2 elements doesn't make a pair. So you need to check that it has exactly 2 elements in order for it to be a pair :)

Reply:
Noted! thanks